### PR TITLE
fixing process error traceback

### PR DESCRIPTION
**Pull request by kurtbrose**

### CODE REVIEW

File: src/seashore/tests/test_shell.py [hunk 1]
```diff
@@ -114,6 +115,18 @@ def test_repr(self):
         self.assertIn('myout', stringified)
         self.assertIn('myerr', stringified)
 
+    def test_trace(self):
+        """traceback has args"""
+        try:
+            raise shell.ProcessError(13, "myout", "myerr")
+        except shell.ProcessError:
+            trace = traceback.format_exc()
```
Comment (moshez) [inline src/seashore/tests/test_shell.py]: can you add an

```
    else:
            self.assertTrue(False, "did not raise an error")
```
here to make sure the traceback is raised.

File: src/seashore/tests/test_shell.py [hunk 2]
```diff
@@ -114,6 +115,18 @@ def test_repr(self):
...
+        stringified = trace.split('\n')[-2]
```
Comment (moshez) [inline src/seashore/tests/test_shell.py]: splitlines?

File: src/seashore/tests/test_shell.py [hunk 3]
```diff
@@ -114,6 +115,18 @@ def test_repr(self):
         self.assertIn('myout', stringified)
         self.assertIn('myerr', stringified)
 
+    def test_trace(self):
+        """traceback has args"""
+        try:
+            raise shell.ProcessError(13, "myout", "myerr")
```
Comment (moshez) [inline src/seashore/tests/test_shell.py]: can you use "more unique" strings for myout/myerr, to make sure the check is not false positived?

like, maybe "my special output string".

**Review by moshez — APPROVED** [review]
Made some comments, nothing blocking.

Please address any that are useful and merge.

File: src/seashore/tests/test_shell.py (hunk 3 above)
Comment (kurtbrose) [reply src/seashore/tests/test_shell.py]: sure, probably want to do the same for test_repr as well then?

File: src/seashore/tests/test_shell.py (hunk 1 above)
Comment (kurtbrose) [reply src/seashore/tests/test_shell.py]: good idea

File: src/seashore/tests/test_shell.py (hunk 2 above)
Comment (kurtbrose) [reply src/seashore/tests/test_shell.py]: good idea

File: src/seashore/tests/test_shell.py (hunk 1 above)
Comment (kurtbrose) [reply src/seashore/tests/test_shell.py]: W:125,12: Redundant use of assertTrue with constant value False (redundant-unittest-assert)

**Pull request merged**